spyder/app/mainwindow.py

```diff
@@ -261,6 +266,7 @@ def __init__(self, options=None):
 
         qapp = QApplication.instance()
         if PYQT5:
+            # Enabling scpaling for high dpi
             qapp.setAttribute(Qt.AA_UseHighDpiPixmaps)

```

Comment:
@ccordoba12 should we also affect this setting with the new option that was added, or it is really not needed?

@SylvainCorlay ?